pyaedt/icepak.py

```diff
@@ -3672,8 +3672,9 @@ def assign_source(
 
         Parameters
         ----------
-        assignment : int or str
-            If int, Face ID. If str, object name.
+        assignment : int or str or list
+            If int, Face ID. If str, object name. It is also possible to provide a list of Face IDs
+            and object names.
```

Comment:
```suggestion
            If an integer is provided, it must be a face ID. If a string is provided, it must
            be an object name. It is also possible to provide a list of face IDs
            and object names.
```
Can the list contain both face IDs and object names? If not, change to ",,, a list of face IDs or object names."